politeiawww/cmd/politeiawwwcli/commands/getdccs.go

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) 2017-2019 The Decred developers
+// Use of this source code is governed by an ISC
+// license that can be found in the LICENSE file.
+
+package commands
+
+import (
+	"fmt"
+
+	v1 "github.com/decred/politeia/politeiawww/api/cms/v1"
+)
+
+// GetDCCsCmd gets all dccs by status.
+type GetDCCsCmd struct {
+	Args struct {
+		Status int `long:"status"`
+	}
+}
+
+// Execute executes the get dccs command.
+func (cmd *GetDCCsCmd) Execute(args []string) error {
```

Comment:
I'm getting a 500 when I run the command:

`politeiawww getdccs` (no status specified)

With the www error being:
```
2019-08-30 15:31:41.015 [ERR] PWWW: 127.0.0.1:55628 POST /v1/dcc HTTP/1.1 Internal error 1567200701: handleGetDCCs: processGetDCCs: record not found
```

Reply:
Hrm seems to be responding properly for me.  Maybe your db got out of sync with d somehow during testing.  But seems as though there is a DCC in the db that isn't being found in the d record cache.  I'll do some further testing to try and replicate the issue.

Reply:
Maybe I just didn't rebuild the cache or something.